### PR TITLE
chore: fix typo in comment

### PR DESCRIPTION
**Pull request by li-jia-nan**
## Summary

Just two typo were fixed:

1. I think the `command+a` shortcut should not be abbreviated, as this makes the meaning unclear
2. I think `onFocus` event should be `called` instead of `call`

## How did you test this change?

Just fix typo, no need to add test case. thx.

### CODE REVIEW

File: packages/react-dom/src/events/plugins/__tests__/SimpleEventPlugin-test.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ describe('SimpleEventPlugin', function () {
             ref={el => (button = el)}
             onFocus={this.increment}
             onClick={() => {
-              // The focus call synchronously dispatches a nested event. All of
+              // The focus called synchronously dispatches a nested event. All of
```
Comment (rickhanlonii) [inline packages/react-dom/src/events/plugins/__tests__/SimpleEventPlugin-test.js:211]: This is referencing the "call" to "focus" to it's correct as-is.

**Review by rickhanlonii — COMMENTED** [review]
Thanks for the suggestions, but I think the existing verbiage is correct.

File: packages/react-dom/src/events/plugins/__tests__/SelectEventPlugin-test.js [hunk 2]
```diff
@@ -188,7 +188,7 @@ describe('SelectEventPlugin', () => {
     // Make sure the event was not called before we emit the selection change event
     expect(onSelect).toHaveBeenCalledTimes(0);
 
-    // This is dispatched e.g. when using CMD+a on macOS
+    // This is dispatched e.g. when using `command+a` on macOS
```
[the remark this answers is not in the document]
Comment (li-jia-nan) [reply packages/react-dom/src/events/plugins/__tests__/SelectEventPlugin-test.js:191]: Thank you for your patience in replying

File: packages/react-dom/src/events/plugins/__tests__/SimpleEventPlugin-test.js (hunk 1 above)
Comment (li-jia-nan) [reply packages/react-dom/src/events/plugins/__tests__/SimpleEventPlugin-test.js:211]: Thank you for your patience in explaining and replying

**Pull request closed without merging**